src/main/services/i18n/index.ts

```diff
@@ -4,9 +4,11 @@ import i18next from 'i18next'
 import Backend from 'i18next-fs-backend'
 import { store } from '../../store'
 
+const lng = store.preferences.get('language')
+
 i18next.use(Backend).init({
   fallbackLng: 'en',
-  lng: store.preferences.get('language'),
+  lng: lng,
```

Comment:
No need to duplicate if prop is the same as the name of a local variable

Reply:
I get it, I tried it and it works. 😀

Reply:
Use just single `lng`

Reply:
Do you mean to change a variable name?

Reply:
Thanks, I see.